src/vs/workbench/api/common/extHostExtensionService.ts

```diff
@@ -7,7 +7,7 @@ import * as nls from 'vs/nls';
 import * as path from 'vs/base/common/path';
 import * as performance from 'vs/base/common/performance';
 import { originalFSPath, joinPath, extUriBiasedIgnorePathCase } from 'vs/base/common/resources';
-import { asPromise, Barrier, IntervalTimer, timeout } from 'vs/base/common/async';
+import { asPromise, Barrier, IntervalTimer, runWhenIdle, timeout } from 'vs/base/common/async';
```

Comment:
Using `runWhenIdle` isn't the best choice here. The truth about it is that it shouldn't be in `/common/` because it only works in a brower/window context. This is on me and I need to clean this up. This usage (in node, worker) will always fallback to `setTimeout0` so I recommend we do just that.